### ISSUE TITLE
Improve the consumability of the data returned by serviceA (and serviceB)

### ISSUE THREAD

**Issue by tevans78** [eclipse/microprofile-service-mesh-service-a#26]
Currently serviceA and serviceB both return a common ServiceData object which is serialized as json.
There are two problems with this;
1. ServiceA and ServiceB should actually return different data; they are different services
2. Even when formatted, the resulting nested json object is hard to read and understand

So the two services should return different data and then that data should be presented in a more readable way.

This issue is not to produce a flashy user interface, just to make the existing data readable (probably with some simple HTML).

**Comment by pilhuhn** [comment]
Yes, return some HTML for service A to better show who is saying what and when.
